decider-bcl2fastq/src/main/java/ca/on/oicr/pde/deciders/Bcl2fastqDecider.java

```diff
@@ -687,6 +674,24 @@ public void setConfig(Map<String, String> config) {
             }
             LaneProvenance lp = Iterables.getOnlyElement(laneNameToLaneProvenance.get(laneName)).getProvenance();
 
+            //expect one sample provenance provider per lane - or no samples in lane
+            List<ProvenanceWithProvider<SampleProvenance>> sps = laneNameToSampleProvenance.get(laneName);
+
+            if (sps.isEmpty()) {
+                laneValidationErrors.add(String.format("Sample provenance count = [%s], expected 1 or more.", sps.size()));
```

Comment:
Should this `continue`?

Reply:
Lane validation errors are collected and returned/printed once all validation rules are applied.